libs/klv/src/test/java/org/codice/alliance/libs/klv/TestMissionIdKlvProcessor.java

```diff
@@ -31,20 +32,19 @@
     public void test() {
 
         String id1 = "ID1";
-        String id2 = "ID2";
 
         ArgumentCaptor<Attribute> argumentCaptor =
                 KlvUtilities.testKlvProcessor(new MissionIdKlvProcessor(),
                         Stanag4609TransportStreamParser.MISSION_ID,
-                        Arrays.asList(id1, id2, id1, id2));
+                        Arrays.asList(id1, id1));
 
         assertThat(argumentCaptor.getValue()
                 .getName(), is(AttributeNameConstants.MISSION_ID));
         assertThat(argumentCaptor.getValue()
```

Comment:
Couldn't these two assertions be combined into `assertThat(argumentCaptor.getValue().getValues(), is(Collections.singletonList(id1))`?